client/slashing.go

```diff
@@ -0,0 +1,38 @@
+package client
+
+import (
+	"context"
+
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	querytypes "github.com/cosmos/cosmos-sdk/types/query"
+	"github.com/cosmos/cosmos-sdk/x/slashing/types"
+)
+
+func (cc *ChainClient) QuerySlashingSigningInfo(publicKey cryptotypes.PubKey) (*types.QuerySigningInfoResponse, error) {
+	params := &types.QuerySigningInfoRequest{ConsAddress: sdk.ConsAddress(publicKey.Address()).String()}
```

Comment:
Also we can't use that sdk.ConsAddress function. That will the global singleton. Need to write up something like the stuff in `client/address.go` to call the underlying bech32 rendering functions and use `cc.EncodeConsAddress`

Reply:
`address.String()` functions are a no-no in lens. This is prob the number one undocumented assumption in the lib